zerver/forms.py

```diff
@@ -378,7 +378,8 @@ def save(
             )
             if active_accounts_in_other_realms:
                 context["active_accounts_in_other_realms"] = active_accounts_in_other_realms
-            language = request.LANGUAGE_CODE
+            language = getattr(request, "LANGUAGE_CODE")
+            assert isinstance(language, str)
```

Comment:
Maybe rather than duplicating this, we should put a function in `zerver/lib/request.py` that's `get_request_language_cide(request)`?  That feels like it might be more readable.